includes/base/controls-stack.php

```diff
@@ -1984,11 +1984,29 @@ public function print_template() {
 	}
 
 	/**
-	 *
 	 * @since 3.6.0
 	 * @access public
+	 *
+	 * @deprecated 3.8.0 Use `::on_import_update_ids()` instead.
 	 */
 	public static function on_import_replace_dynamic_content( $config, $map_old_new_post_ids ) {
+		Plugin::$instance->modules_manager->get_modules( 'dev-tools' )->deprecation->deprecated_function( __METHOD__, '3.8.0', __CLASS__ . '::on_import_update_ids()' );
+
+		return $config;
+	}
+
+	/**
+	 * On import update post and term IDs.
+	 *
+	 * @since 3.8.0
+	 *
+	 * @param array $config
+	 * @param array $data
+	 * @param array|null $controls
```

Comment:
Maybe explain the parameters and their uses?

Reply:
Done.